src/metabase/api/search.clj

```diff
@@ -272,6 +272,11 @@
 (s/defmethod search-query-for-model "collection"
   [model search-ctx :- SearchContext]
   (-> (base-query-for-model model search-ctx)
+      ;; TODO: get this working
```

Comment:
this comment needs to go. Either its already done or needs to be done.

Reply:
Removed the comment.

It's a bit weird, but the query is in search config:
https://github.com/metabase/metabase/pull/21171/files#diff-52b1b2f700169bc51d0f78761bfb84a70a7f1bdec8137889e2b089f30a22875eR158